### DISCUSSION TITLE
Observable cells not working when using `quarto render`

### DISCUSSION THREAD

**Discussion by adityam** [quarto-dev/quarto-cli#5277] [Q&A] [answered]
I am using `obs` cells, and they work well when I preview the page using `quarto preview`. However, when I generate standalone HTML output using `quarto render`, only the `obs` code is shown, not the output. 

Do I need any options in `_quarto.yml` to enable embedding of `obs` output. 

MNWE:

    ## Seattle Precipitation by Day (2012 to 2016)

    ```{ojs}
    data = FileAttachment("seattle-weather.csv")
      .csv({typed: true})
      
    Plot.plot({
      width: 800, height: 500, padding: 0,
      color: { scheme: "blues", type: "sqrt"},
      y: { tickFormat: i => "JFMAMJJASOND"[i] },
      marks: [
	Plot.cell(data, Plot.group({fill: "mean"}, {
	  x: d => d.date.getUTCDate(),
	  y: d => d.date.getUTCMonth(),
	  fill: "precipitation", 
	  inset: 0.5
	}))
      ]
    })
    ```

where `seattle-weather.csv` is downloaded from [here](https://raw.githubusercontent.com/vega/vega/main/docs/data/seattle-weather.csv)

**Comment by cscheid** [comment, marked as answer]
I suspect you're double-clicking on the HTML file instead of running it from a server.

**Reply by adityam** [reply]
Ah interesting. I created a test repo for github pages, and it does render correctly there. 

Thanks!

**Reply by cscheid** [reply]
A _local_ server also works. If you have python on your machine, you can type `python -m http.server` and that will start a web server at localhost:8000 that serves the contents of that directory.

This limitation is by design: a number of javascript features don't work in the file:/// protocol for security reasons (a malicious webpage could otherwise open any files on your computer).
